week-1-practise/level-2/2-format-currency.js

```diff
@@ -2,24 +2,21 @@
   CURRENCY FORMATTING
   ===================
   The business has informed you that prices must have 2 decimal places
-  They must also start with the currency symbol
+  They must also start with the currency symbol 
   Write a function that transforms numbers into the format £0.00
 */
 
-function formatCurrency() {}
+formatCurrency = price => "£" + price.toFixed(2);
 
 /* DO NOT EDIT BELOW THIS LINE */
 /* --------------------------- */
-
-function calculateSalesTax(price) {
-  return price * 0.2;
-}
+calculateSalesTax = price => price * 0.2;
```

Comment:
line 13 assume it it's inside a comment
//calculateSalesTax = price => price * 0.2;